localstack/services/kms/models.py

```diff
@@ -129,11 +129,14 @@ def deserialize_ciphertext_blob(ciphertext_blob: bytes) -> Ciphertext:
 
 
 def _serialize_encryption_context(encryption_context: EncryptionContextType) -> bytes:
```

Comment:
```suggestion
def _serialize_encryption_context(encryption_context: Optional[EncryptionContextType]) -> bytes:
```

because the default arg is now set to `None` in `encrypt` method